assets/scripts/app.js

```diff
@@ -0,0 +1,42 @@
+//Importing HTML Elements
+//Defining getElementById, querySelector & querySelectorAll through function for code shortening
+function getByid(param) {
+    return document.getElementById(`${param}`);
+}
+
+function getSelector(param) {
+    return document.querySelector(`${param}`);
```

Comment:
```suggestion
    return document.querySelector(param);
```
zar ne radi ako stavis samo ovo?